### PR TITLE
Cleanup OpenLoco.cpp

### PR DESCRIPTION
**Pull request by duncanspumpkin**
Deleting some dead code. Implementing some simple setup funcs.

### CODE REVIEW

File: src/OpenLoco/OpenLoco.cpp [hunk 1]
```diff
@@ -1256,18 +1249,16 @@ namespace OpenLoco
 
             resetCmdline();
             registerHooks();
-            if (sub_4054B9())
```
Comment (LeftofZen) [inline src/OpenLoco/OpenLoco.cpp:1259]: what did this deleted function do?

Comment (duncanspumpkin) [reply src/OpenLoco/OpenLoco.cpp:1259]: Registered the wndclass which is handled by sdl in our code

File: src/OpenLoco/Graphics/Gfx.cpp [hunk 2]
```diff
@@ -240,6 +240,46 @@ namespace OpenLoco::Gfx
...
+                // TODO: Use an indirection to convert from character to imageId
+                auto* element = getG1Element(ImageIds::characters_medium_normal_space + i + font.offset);
+                if (element == nullptr)
+                {
+                    continue;
+                }
+                auto width = element->width + font.widthFudge;
+                // Characters from 123 to 150 are unused
+                // Unsure why this zeros it out though since a negative width isn't an issue
+                if (c >= '{' && c <= '\u0096')
```
Comment (AaronVanGeffen) [inline src/OpenLoco/Graphics/Gfx.cpp:273]: Given CI errors, perhaps we could just use:
```suggestion
                if (c >= '{' && c <= '\x96')
```

**Review by AaronVanGeffen — APPROVED** [review]
Looks good to me. Gave the PR a quick test and found no issues.

**Pull request merged**